build/configs/rtl8730e/loadable_apps_16M/defconfig

```diff
@@ -290,7 +294,7 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="48,52,12,400,1844,4808,384,1844,4808,384,512,1280,8,"
+CONFIG_FLASH_PART_SIZE="48,52,12,400,1844,4936,384,1844,4936,384,512,1024,8,"
```

Comment:
Please check recently partition size

https://github.com/Samsung/TizenRT/pull/5964/commits/373135ac024bb4d4aef03ef4fc65e0db30e1226d

Reply:
I think, This sync is duplicating, please check what is correct...

Reply:
I understand your concern. There could be some confusion about the latest update partition sizes for use.
I'll check and update